tests/jenkins/jobs/ProcessAndRunOsdIntegTest_Jenkinsfile

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+pipeline {
+    agent none
+    stages {
+        stage('integ-test') {
+            steps {
+                script {
+                    processAndRunOsdIntegTest(
+                        localComponent: 'OpenSearch',
```

Comment:
This should fail right? Since the lib as the name suggests only tests OSD?

Reply:
This is just a dummy component name to check component without any ci-group.

Reply:
Done